api/src/Entity/TopBook.php

```diff
@@ -0,0 +1,123 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Entity;
+
+use ApiPlatform\Core\Annotation\ApiProperty;
+use ApiPlatform\Core\Annotation\ApiResource;
+
+/**
+ * This entity represents a "most borrowed book" in a given a given French library.
+ * It is loaded from a CSV file. It's purpose it to show how to expose data coming
+ * from external data sources through API Platform.
+ *
+ * @see https://www.data.gouv.fr/fr/datasets/top-100-romans-adultes-de-science-fiction-et-fantastiques/
+ * @see https://www.data.gouv.fr/fr/datasets/r/a2b09081-6cc4-4fdb-bd73-35484014c89c
+ * @see /data/top-100-novel-sci-fi-fr.csv
+ *
+ * @ApiResource(
+ *   collectionOperations={"get"},
+ *   itemOperations={"get"}
+ * )
+ */
+class TopBook
+{
+    /**
+     * This ID is the "rank" to the top book, from 1 to 100.
+     *
+     * @ApiProperty(identifier=true)
+     */
+    private int $id;
+
+    private string $title;
```

Comment:
Please set a default value for PHP 7.4 typehint:
```php
private string $title = '';
```
or
```php
private ?string $title = null;
```
(same for other properties)